glean-core/metrics.yaml

```diff
@@ -238,8 +238,83 @@ glean.internal.metrics:
       - glean-team@mozilla.com
     expires: never
 
-  ping_reason:
+  seq:
     type: string
+    lifetime: ping
+    send_in_pings:
+      - glean_internal_info
```

Comment:
It's too bad we have `glean_internal_info` and couldn't call it `glean_ping_info` (as we have for `glean_client_info`).  Is that enough of an implementation detail that we could change it without too much pain?  That would make things more obvious for users over at the probe_scraper / Glean dictionary end of things...

Reply:
We do persist `experiments` using this. The `experiments` annotation is `application` only so we _could_ migrate away from that over to `glean_ping_info` with relative ease. I'd rather do that in a follow-up though.

Reply:
I spoke to soon: we also store the start_time with a `user` lifetime, so it's a bit more involved.